.github/workflows/priority-support-validation-prompt.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     steps:
       - name: Find Comment
-        uses: peter-evans/find-comment@v2 # v2.4.0
+        uses: peter-evans/find-comment@a54c31d7fa095754bfef525c0c8e5e5674c4b4b1 # v2
```

Comment:
@DanailH Doesn't this change look somewhat strange?
Maybe it's because this is the first time this dep is being bumped and it is being changed from a version to a hash and we should manually fix it to the exact version this time? 🤔 🤷

Reply:
```suggestion
        uses: peter-evans/find-comment@a54c31d7fa095754bfef525c0c8e5e5674c4b4b1 # v2.4.0
```

Reply:
Same happened in the core. It's safe to merge ;)